src/Computation/Proposition.cs

```diff
@@ -3,14 +3,14 @@
 
 namespace Symbolica.Computation;
 
-internal sealed class SymbolicProposition : IProposition
```

Comment:
So we're just happy to create these unconditionally of whether it's symbolic or not because something else will take care of ignoring the constant ones?

Reply:
Yeah, we should discuss this more generally. I think it is useful in both of our approaches to reading/writing etc, but I could be wrong.